api/main.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+import openai
+from fastapi import FastAPI, HTTPException, Query, status
+
+from api.openai_client import client
+
+app = FastAPI(root_path="/api")
+
+
+@app.get("/teamDescription")
+def get_team_description(team_name: str = Query(alias="teamName")) -> dict[str, str]:
```

Comment:
v deklaraci funkce mate `dict[str, str]`, ale na vystupu vracite string, tak je to nekonzistentni.